app-crypt/openpgp-keys-voikko/openpgp-keys-voikko-20201013.ebuild

```diff
@@ -0,0 +1,22 @@
+# Copyright 2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="OpenPGP keys used to sign voikko* packages"
+HOMEPAGE="https://voikko.puimula.org/"
+SRC_URI="https://www.puimula.org/voikko-sources/voikko-fi/voikko-fi-2.3.tar.gz.asc
```

Comment:
I'm confused. Are those keys or signatures?

Reply:
These are indeed signatures. The pgp public key is plaintexted in their website but I can't find a downloadable link to it. Gonna hunt for it somewhere, like SKS next.

Reply:
@mgorny how do I get the block from http://keys.gnupg.net, this'd be the full address:
http://keys.gnupg.net/pks/lookup?op=get&search=0x3D309B604AE3942E
```
SRC_URI="http://keys.gnupg.net/pks/lookup?op=get&search=0x3D309B604AE3942E" -> 4AE3942E.asc"
```
and then sed/awk/grep to remove everything outside `-----BEGIN -> END----etc`? Or do I just simply put it in `files/4AE3942E.asc`? and use that?

Reply:
Either works for me, or inlining it in ebuild.